<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTwoEditDlg</class>
 <widget class="QDialog" name="QTwoEditDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>136</height>
   </rect>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Line1</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit1</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Line2</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit2</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelHelp">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBtn">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit1</tabstop>
  <tabstop>lineEdit2</tabstop>
  <tabstop>OKBtn</tabstop>
  <tabstop>CancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>OKBtn</sender>
   <signal>clicked()</signal>
   <receiver>QTwoEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>QTwoEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
